### ISSUE TITLE
Test on validation set with a pretrained model

### ISSUE THREAD

**Issue by SeokjuLee** [JiawangBian/SC-SfMLearner-Release#12]
Hi, when I start training with the pre-trained models (k_depth.tar & k_pose.tar), the initial Abs_Rel error on KITTI raw validation set begins with 0.09781. However, with the scratch training, the Abs_Rel error looks like below.
![image](https://user-images.githubusercontent.com/17178662/65140124-d9e9f600-da48-11e9-87c8-cae322329079.png)
Is the Abs_Rel error with this scratch training eventually (200 epochs as the paper?) converge to the value of the pre-trained models?

**Comment by JiawangBian** [comment]
maybe not, but their results on test sets (697 images) should be similar.

1. As the train/validation split is different (it is randomly generated), so the k_depth is partially trained on your validation sets. Therefore, its results look better.

2. k_depth and k_pose are actually not coupled. k_pose is trained on kitti odometry 00-08 sequences. I am not sure that what will happen if combining them.

3. Based on my experience, your validation error looks good in this stage (note that the results reported in paper are about 0.137, while your validation error has been lower than 0.15 after 10 epoches).

**Comment by SeokjuLee** [comment]
Thank you for your replies. One more question here. So did you also split the train/val set with the same ratio when you make the pre-trained model of DispResNet? I am curious this because you fixed the random seed (np.random.seed(8964)) in the 'data/prepare_train_data.py'. I thought you made the pre-trained model based on this random seed.

**Comment by JiawangBian** [comment]
the ratio is same。see randomness in issue#1

**Comment by SeokjuLee** [comment]
I got it Thanks!
